classes/activities/class.circle-of-support.php

```diff
@@ -0,0 +1,72 @@
+<?php
+
+class CircleOfSupport
+{
+    private $userId;
+    private $versions;
+
+    const META_KEY = 'circle_of_support';
```

Comment:
Please prepend this with `tyde_` so it doesn't potentially conflict with other meta keys that might be named similarly by other plugins.